client/package.json

```diff
@@ -160,6 +160,9 @@
 		"utility-types": "3.10.0",
 		"uuid": "7.0.2",
 		"valid-url": "1.0.9",
+		"webpack": "^4.42.1",
+		"webpack-dev-middleware": "^3.7.2",
+		"webpack-hot-middleware": "^2.25.0",
```

Comment:
Do we really need these here? We normally treat them as `devDependencies`, and in a monorepo setup -- such as Calypso's -- they should be in the root `package.json`. And they already are: :slightly_smiling_face: 

https://github.com/Automattic/wp-calypso/blob/db103bd3401d4fe4324d3d9d1e3743b0071533e5/package.json#L291-L295

Reply:
Thanks for pointing it out! I added those because eslint was not picking them up from the root package.json. As it appears, this conflicts with our monorepo rules, so I removed those and simply disabled the rule for those specific require statements in `client/server/bundler/index.js`.

Reply:
Thank you! Yep, that's what we ususally do to work around that rule :+1: